Gopkg.toml

```diff
@@ -47,8 +47,8 @@ required = [
   name = "github.com/giantswarm/backoff"
 
 [[constraint]]
-  branch = "k8s-1.15"
-  name = "github.com/giantswarm/e2esetup"
+  branch = "master"
+  name = "github.com/giantswarm/clientk8s"
```

Comment:
Why this repo has inverted name? So far we had `helmclient` and not `clienthelm`.

Reply:
Because this is what Theo suggested and I did not mind either way.